City-of-Helsinki/helsinki-paatokset
# Pp 219 policymaker listing

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-219

**What was done**

Created a block that renders to a page that has /paattajat as its path. Made only the browsing part of the Päättäjät search page in this ticket and the search should be done in another ticket. The styles created for this can be reused for search results.

Images for Kaupunginhallitus and valtuusto are missing but there is a ready space for them.

**How to test**

Checkout branch and run `drush cim -y; drush cr`. Make sure you have policymaker and trustee nodes in your local environment.

Create a landing page that is named "Päättäjät" and make sure the url is /paattajat. Add a hero section is you want the header to look similar to the design.

The new block should now appear on the page. Check the styles in desktop and in mobile. Keep in mind that the container-class has broken widths because of the webpack bug.  This also affects the height of the "images" for kauounginhallitus and valtuusto.
 
Check code changes. Could something in paatokset_ahjo_api.module file be done better?
